src/sidebar/services/frame-sync.ts

```diff
@@ -672,6 +676,34 @@ export class FrameSyncService {
     guest.call('scrollToAnnotation', ann.$tag);
   }
 
+  /** Request a thumbnail from an annotated region of the document. */
+  async requestThumbnail(
+    tag: string,
```

Comment:
What does `tag` represent here?

Reply:
The annotation's `$tag` property. I've updated the docs.